web_src/js/features/common-global.js

```diff
@@ -167,6 +167,21 @@ export function initGlobalDropzone() {
           file.uuid = data.uuid;
           const input = $(`<input id="${data.uuid}" name="files" type="hidden">`).val(data.uuid);
           $dropzone.find('.files').append(input);
+          // Create a "Copy Link" element, to conveniently copy the image
+          // or file link as Markdown to the clipboard
+          const copyLinkElement = document.createElement('a');
+          copyLinkElement.className = 'dz-remove';
+          copyLinkElement.href = '#';
+          copyLinkElement.innerHTML = '<i class="fa fa-copy"></i> Copy link';
```

Comment:
FWIW, we shouldn't use FontAwesome in new code any more, but SVG octicon.

Reply:
And for me, personally I would never put a `dz-remove` class on the `Copy` button.

Reply:
Since this was merged, should I do another PR addressing this feedback?

Reply:
Yeah it would be worth doing so if you can. Thanks!